lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -172,7 +197,7 @@ def main():
     argument_spec = meraki_argument_spec()
     argument_spec.update(
         net_id=dict(type='str'),
-        type=dict(type='str', choices=['wireless', 'switch', 'appliance', 'combined'], aliases=['net_type']),
+        type=dict(type='list', choices=['wireless', 'switch', 'appliance'], aliases=['net_type']),
```

Comment:
This is not backward compatible. Fine if you want to continue regardless :-)

Reply:
If I'm going to do it, now is probably the time before many people are using these modules.